src/main/java/picard/util/AlleleSubsettingUtils.java

```diff
@@ -176,10 +176,15 @@ public static GenotypesContext subsetAlleles(final GenotypesContext originalGs,
                 }
             }
 
-            final boolean useNewLikelihoods = newPLs != null;
-            final GenotypeBuilder gb = useNewLikelihoods ? new GenotypeBuilder(g).PL(newPLs).log10PError(newLog10GQ) : new GenotypeBuilder(g).noPL().noGQ();
-            GenotypeLikelihoods.GenotypeLikelihoodsAllelePair allelePair = GenotypeLikelihoods.getAllelePair(MathUtil.indexOfMin(newPLs));
-            gb.alleles(Stream.of(allelePair.alleleIndex1, allelePair.alleleIndex2).map(allelesToKeep::get).collect(Collectors.toList()));
+            final GenotypeBuilder gb;
+            if (newPLs == null) {
+                gb = new GenotypeBuilder(g).noPL().noGQ().alleles(Arrays.asList(Allele.NO_CALL, Allele.NO_CALL));
```

Comment:
This list already defined as `GATKVariantContextUtils.NO_CALL_ALLELES`.  Although you might want to change the name to `DIPLOID_NO_CALL_ALLELES`.

Reply:
we're in picard...yes?

Reply:
though, it's a good idea to make one copy of that list in picard as well....note taken.

Reply:
I guess we are in Picard, but yeah, I think it belongs as a `public static` somewhere.